packages/fetch-extension/src/pages/notiphy-notification/notification-organizations/index.tsx

```diff
@@ -64,15 +62,13 @@ export const NotificationOrganizations: FunctionComponent = observer(() => {
         if (organisations.length == 0) {
           /// Updating the pre-selected orgs.
           fetchFollowedOrganisations(accountInfo.bech32Address).then(
-            (followOrganisationList: NotyphiOrganisation[]) => {
+            (followOrganisationList: any) => {
```

Comment:
same type issue here??

Reply:
it was the same earlier but with redux we didnt use to get a type issue , @sh-wallet i hvnt actually modified it as such just to resolve the error i did it the follwoing way, please check if you can change the organisation type at the store end as this was your design you may be aware of its work